mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/MasterMain.java

```diff
@@ -103,17 +96,14 @@ public class MasterMain implements Service {
     private final ServiceLifecycle mantisServices = new ServiceLifecycle();
     private final AtomicBoolean shutdownInitiated = new AtomicBoolean(false);
     private KeyValueBasedPersistenceProvider storageProvider;
-    private CountDownLatch blockUntilShutdown = new CountDownLatch(1);
-    private volatile CuratorService curatorService = null;
+    private final CountDownLatch blockUntilShutdown = new CountDownLatch(1);
     private MasterConfiguration config;
     private ILeadershipManager leadershipManager;
-    private final MantisPropertiesLoader dynamicPropertiesLoader;
```

Comment:
there are cleanups where I removed things that appear dead. I am happy to start reverting these.